sky/serve/service_spec.py

```diff
@@ -23,8 +23,12 @@ def __init__(
         max_replicas: Optional[int] = None,
         qps_upper_threshold: Optional[float] = None,
         qps_lower_threshold: Optional[float] = None,
```

Comment:
How about we move the deprecated replicas to the end? If no one is using this arguments, maybe we can directly remove these arguments for code simplicity.
```suggestion
       # Deprecated: the two arguments below is replaced by the
       # target_qps_per_replica
       # TODO(ziming): remove this in version 0.6.0
        qps_upper_threshold: Optional[float] = None,
        qps_lower_threshold: Optional[float] = None,
```